apiml-common/src/test/java/org/zowe/apiml/product/web/HttpConfigTest.java

```diff
@@ -49,6 +63,57 @@ void whenKeystore_thenDoNothing() {
             assertNull(ReflectionTestUtils.getField(httpConfig, "trustStorePassword"));
         }
 
+        @Test
+        void whenVerifySsl_thenNoThrow() {
+            HttpConfig httpConfig = new HttpConfig();
+            ReflectionTestUtils.setField(httpConfig, "maxConnectionsPerRoute", 10);
+            ReflectionTestUtils.setField(httpConfig, "maxTotalConnections", 100);
+
+            ReflectionTestUtils.setField(httpConfig, "keyStore", pathFromRepository("keystore/localhost/localhost.keystore.p12"));
```

Comment:
Maybe you can create a private method `setupHttpConfig(HttpConfig httpConfig, String keyAlias)` to avoid duplication of these sets